BaseLib/TemplateLogogFormatterSuppressedGCC-impl.h

```diff
@@ -16,10 +16,7 @@ ::TemplateLogogFormatterSuppressedGCC(MPI_Comm mpi_comm)
 {
     int rank = 0;
     MPI_Comm_rank(mpi_comm, &rank);
-    int size = 0;
-    MPI_Comm_size(mpi_comm, &size);
-    int digits = size/10 + 1;
-    _str_mpi_rank = "rank " + padLeft(std::to_string(rank), digits) + "| ";
+    _str_mpi_rank = "[" + std::to_string(rank) + "] ";
```

Comment:
The former code didn't work?

Reply:
I think the padding was not computed correctly. Furthermore, my aim was to reduce the size of the message as much as possible but containing the relevant information.